charts/dapr/charts/dapr_operator/templates/dapr_operator_deployment.yaml

```diff
@@ -67,6 +67,18 @@ spec:
         prometheus.io/path: "/"
 {{- end }}
     spec:
+{{- if eq .Values.global.ha.enabled true }}
```

Comment:
Not sure if it's included by accident, but it looks irrelevant to this change.

Reply:
I fixed it, it's included by accident.